### PR TITLE
Functionality for refreshing all sites for a particular tenant

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**
Introduces functionality for refreshing all sites for a tenant

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-1040](https://airqoteam.atlassian.net/browse/PLAT-1040)

**_HOW DO I TEST OUT THIS PR?_**
1. Setup your environment using instructions on the [readme.md file](https://github.com/airqo-platform/AirQo-api/tree/staging/src/data-mgt/python/devices-tranformation#setup-your-environment)

2. Trigger the functions 
```python
python main.py refresh_sites
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
Get sites endpoint. All sites must be refreshed after task completion

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

File: src/data-mgt/python/devices-tranformation/airqoApi.py [hunk 1]
```diff
@@ -88,6 +88,10 @@ def update_sites(self, updated_sites):
             response = self.__request("devices/sites", params, site, "put")
             print(response)
 
+    def refresh_site(self, params):
```
Comment (Baalmart) [inline src/data-mgt/python/devices-tranformation/airqoApi.py:91]: this endpoint does not automatically update the `search_name` and `location_name`, you need to explicitly update those two fields using the **Site update** endpoint.

The refresh endpoint only refreshes/updates weather and location data, and NOTHING else. @NoahNsimbe

**Review by Baalmart — COMMENTED** [review]
Sounds good, just left a comment for your review. @NoahNsimbe

File: src/data-mgt/python/devices-tranformation/airqoApi.py (hunk 1 above)
Comment (NoahNsimbe) [reply src/data-mgt/python/devices-tranformation/airqoApi.py:91]: This functionality doesn't update `search_name` and `location_name`. It just refreshes all sites by calling the refresh endpoint as stated in the PR description. To update `search_name` and `location_name`, consider the following documentation. 
https://github.com/airqo-platform/AirQo-api/tree/staging/src/data-mgt/python/devices-tranformation#update-sites-search-names-based-on-csv-file
@Baalmart

**Review by Baalmart — APPROVED** [review]

**Pull request merged**